lte/cloud/go/services/subscriberdb/servicers/subscriberdb_servicer.go

```diff
@@ -89,11 +95,96 @@ func (s *subscriberdbServicer) ListSubscribers(ctx context.Context, req *lte_pro
 		subProto.NetworkId = &protos.NetworkID{Id: networkID}
 		subProtos = append(subProtos, subProto)
 	}
+
 	listRes := &lte_protos.ListSubscribersResponse{
 		Subscribers:   subProtos,
 		NextPageToken: nextToken,
 	}
-	return listRes, nil
+
+	return listResWithDigest(s.flatDigestEnabled, listRes, req, gateway, apnsByName, apnResourcesByAPN), nil
+}
+
+// listResWithDigest returns a ListSubscribersResponse with the fields augmented
+// according to the flat digest logic.
+func listResWithDigest(
```

Comment:
There's a less messy way of doing this -- just have a fn that returns (digest, no_updates) or something like that (and takes into account the feature flag -- which you can get from the method receiver rather than as an arg). Then can place those values into the proto from the upper fn